terms-of-service.md

```diff
@@ -29,11 +29,11 @@ Policy](https://whereby.com/information/tos/privacy-policy/)
 you don’t agree to the Terms, you can’t use Whereby.
 
 We may change these Terms at any time. We keep a historical record of
-all changes to our Terms on GitHub. If a change is material, we’ll let
+all changes to our Terms on [GitHub](https://github.com/whereby/policy). If a change is material, we’ll let
```

Comment:
I could also add a link at the bottom or top of this with a link to this very document on GitHub.

But I'll do that as a separate thing if we want it later :)